js/client.js

```diff
@@ -368,6 +368,18 @@
 						}
 					}
 					this.send('/autojoin ' + autojoinIds.join(','));
+
+					var battleAutojoin = (Tools.prefs('battleautojoin') || '').split(',');
```

Comment:
We don't need a separate `battleAutojoin`; can't we just add it to the normal autojoin list?